spec/smartcar/e2e/user_spec.rb

```diff
@@ -5,23 +5,24 @@
 
 RSpec.describe Smartcar::User do
   subject { Smartcar::User }
-  let(:token) do
+
+  before(:context) do
```

Comment:
The improvements here is that previously using `let` meant it ran the auth flow for every single test case again and again. Now it will run only once for this suite and use the same token